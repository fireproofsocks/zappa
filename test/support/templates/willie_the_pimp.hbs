{{>denomination}} (I can set you straight)
Meet me onna corner boy 'n don't be late