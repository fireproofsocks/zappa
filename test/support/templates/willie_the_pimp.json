{
  "title": "Willie the Pimp",
  "vocals": "Captain Beefheart",
  "length": "00:09:25",
  "released": "1969-10-10",
  "genres": ["blues rock", "hard rock", "jazz rock"]
}