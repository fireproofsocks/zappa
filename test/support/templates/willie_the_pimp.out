Twenny dollah bill (I can set you straight)
Meet me onna corner boy 'n don't be late